modules/ROOT/pages/waf-example.adoc

```diff
@@ -0,0 +1,7 @@
+= WAF Example
+
+In this example, the WAF policy is configured for JSON, and checks the HTTP headers, URI request, parameters, and the request body.
+
+Leaving the *Disable body scanner* option unchecked can be costly in terms of resources (CPU) depending on the size of the request messages. If request body checking is enabled, the CAP policy “Maximum Message Size” should be set to a value no larger than required to prevent attackers from abusing request body checking, and exhausting the resources. 
```

Comment:
unclear who is exhausting the resources, abusing request-body checking or not checking the option.